menas/src/main/scala/za/co/absa/enceladus/menas/services/SchemaRegistryService.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas.services
+
+import java.net.URL
+
+import com.typesafe.config.ConfigFactory
+import org.springframework.beans.factory.annotation.{Autowired, Value}
+import org.springframework.stereotype.Service
+import za.co.absa.enceladus.menas.controllers.SchemaController
+import za.co.absa.enceladus.menas.models.rest.exceptions.RemoteSchemaRetrievalException
+import za.co.absa.enceladus.menas.services.SchemaRegistryService.SchemaResponse
+import za.co.absa.enceladus.menas.utils.SchemaType
+import za.co.absa.enceladus.utils.config.SecureConfig
+
+import scala.io.Source
+import scala.util.control.NonFatal
+
+@Service
+class SchemaRegistryService @Autowired()() {
+
+  @Value("${menas.schemaRegistryBaseUrl}")
+  val schemaRegistryBaseUrl: String = ""
```

Comment:
OK, agreed to change as suggested (the underlying reason: we want to support Menas without a dedicated schema registry running)

Reply:
OK, done. I hope I came up with a solution similar to the one you probably had in mind.

Since the configuration is only on backend, the backend exposes an availability endpoint that signifies what schema-update options are available. Based on this, the UI starts with the by-topicName upload disabled, queries the endpoint to enable/keep disabled the newly added option.

Looks like this:
![schema-registry-disabled](https://user-images.githubusercontent.com/4457378/85307069-b78d7900-b4af-11ea-82a6-f91d35776eaf.png)
![schema-registry-enabled-by-availability](https://user-images.githubusercontent.com/4457378/85307083-bc522d00-b4af-11ea-82dc-24e37c35d5b0.png)

Reply:
I like it. I like the API behind it as well, will give us options for this to be configurable as desired.

Reply:
After a discussion with @Zejnilovic, the API was changed to only include the repository option.